configuration/pih/petl/jobs/datawarnings/aggregrate/source.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
+    warning_type,
+    count(*) as count,
+    NOW() as timestamp
+FROM hivmigration_data_warnings
+GROUP BY warning_type
```

Comment:
This is pretty simple pattern, but it's possible that it could be used for other simple aggregate queries:  note that it uses a group by and also added a timestamp column. "dropAndRecreateTable" has been set to false, so on each run the it inserts a row per unique warning into the target table with the counts of each unique.  So, as data errors are resolved, this should be a way to track the decrease in errors over time.